static/src/stylesheets/components/pasteup-forms/src/_forms.scss

```diff
@@ -273,6 +272,14 @@ category: Common
     }
 }
 
+.submit-input {
+    @include submit-input--behaviour;
+    padding: 11px $gs-gutter/2;
+}
+.submit-input--behaviour {
+    @include submit-input--behaviour;
+}
+
```

Comment:
You could do this without mixins:

``` css
.submit-input,
.submit-input--behaviour {
   /* common styles */
}

.submit-input {
  padding: foo;
}
```

Also, this is not how we usually use BEM. Modifiers are supposed to be used in combination with the block class, e.g. `submit-input submit-input--behaviour`, in order to compose styles.

Perhaps the block class should be all the common styles, and the modifier should just contain the padding? This would mean we have to opt-in to it everywhere else in the codebase though.